services/worker-manager/src/provisioner.js

```diff
@@ -136,6 +137,16 @@ class Provisioner {
         handler: seen,
       });
 
+      // compute the number of instances that have not yet called "registerWorker"
+      await this.Worker.scan({
+        state: Entity.op.equal(this.Worker.states.REQUESTED),
+      }, {
+        handler: worker => {
+          const v = providersByPool.get(worker.workerPoolId);
+          v.requested += worker.capacity;
+        },
+      });
```

Comment:
I thought about that, may I read the `state` property as a normal Javascript object field or does it require a special invocation? This question is because I implemented a second scan.